dahu/core/app/scripts/views/workspace/actions/move.js

```diff
@@ -5,19 +5,63 @@
 define([
     'handlebars',
     'backbone.marionette',
+    // views
+    'views/workspace/actions/action',
     // templates
     'text!templates/views/workspace/actions/move.html'
 ], function(
     Handlebars,
     Marionette,
+    // views
+    ActionView,
     // templates
     moveTemplate
 ) {
-    
+
+    var ExtraParamView = Marionette.ItemView.extend({
+
+        template: Handlebars.default.compile(moveTemplate),
+
+        initialize: function (model) {
+            this.model = model;
+        },
+
+        triggers: {
+            "change .trXChoice": "move:trx:change",
+            "change .trYChoice": "move:try:change"
+        }
+    });
+
     /**
      * Move action view
      */
-    return Marionette.ItemView.extend({
-        template: Handlebars.default.compile(moveTemplate)
+    return ActionView.extend({
+
+        onChildviewMoveTrxChange: function () {
+            var id = "#trXChoice_" + this.model.id;
+            var value = parseInt($(id).val());
+            if (!_.isNaN(value)){
+                this.model.set('trX', value);
+            }
+            else {
+                throw new Exception.IOError("Translation X input is NaN");
+            }
+        },
+
+        onChildviewMoveTryChange: function () {
+            var id = "#trYChoice_" + this.model.id;
+            var value = parseInt($(id).val());
+            if (!_.isNaN(value)){
+                this.model.set('trY',value);
+            }
+            else {
+                throw new Exceptions.IOError("Translation Y input is NaN");
+            }
+        },
+
+        onRender: function () {
```

Comment:
@barraq Doesnt work with 'onShow' neither with 'onBeforeShow'.
'onBeforeShow' will never display extra settings
'onShow' will not display settings on refresh (for trigger change)